README.md

```diff
@@ -37,6 +37,25 @@ On platforms other than Windows, you will need to have [Wine](http://winehq.org)
 * `application-manifest` - String path to a local manifest file to use.
   See [here](https://msdn.microsoft.com/en-us/library/windows/desktop/aa374191.aspx)
   for more details.
+ 
+`version-string` allows you to set the PE file meta keys, such as in the following example:
+
+```
+var options = {
```

Comment:
This block should probably go in an "options example" section (I'm not entirely sure what the section should be titled though), since it's showing more than just an example of `version-string`.